packages/nextra/src/client/components/search.tsx

```diff
@@ -228,7 +228,7 @@ export const Search: FC<SearchProps> = ({
           spellCheck={false}
           className={({ focus }) =>
             cn(
-              'x:rounded-lg x:px-3 x:py-2 x:transition-colors',
+              'x:rounded-lg x:px-3 x:py-2 x:transition',
```

Comment:
I prefer to be explicit in applying transitions, which transition should we apply here in addition to `color`/`bg-color`?